idd/Energy+.idd.in

```diff
@@ -23564,6 +23572,70 @@ AirflowNetwork:Distribution:Linkage,
       \note The zone name is where AirflowNetwork:Distribution:Component:Duct is exposed. Leave this field blank if the duct
       \note conduction loss is ignored.
 
+AirflowNetwork:OccupantVentilationControl,
```

Comment:
@mjwitte If you have time, could you look over the IDD for this object? Maybe you already have. It looks good to me, but your opinion would be very helpful.